2022-RADIUSS-AWS/JupyterNotebook/radiuss-aws-flux.ipynb

```diff
@@ -0,0 +1,944 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e693f2d9-651f-4f58-bf53-62528caa83d9",
+   "metadata": {},
+   "source": [
+    "The output indicates the number of brokers started successfully."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eda1a33c-9f9e-4ba0-a013-e97601f79e41",
+   "metadata": {},
+   "source": [
+    "## Flux uptime\n",
+    "Flux provides an `uptime` utility to display properties of the Flux instance such as state of the current instance, how long it has been running, its size and if scheduling is disabled or stopped. The output shows how long the instance has been up, the instance owner, the instance depth (depth in the Flux hierarchy), and the size of the instance (number of brokers)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6057ce25-d1b3-4cc6-b26a-4b05a1639616",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!flux uptime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dee2d6af-43fa-490e-88e9-10f13e660125",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Submitting Jobs to Flux\n",
+    "## Submission CLI\n",
+    "### `flux mini`: the Minimal Job Submission Tool\n",
+    "\n",
+    "To submit jobs to Flux, you can use the `flux mini` command, which has several sub-commands: `submit`, `run`, `bulksubmit`, `batch`, and `alloc`.  The `flux mini submit` command submits a job to Flux and prints out the jobid. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a5e7d41-1d8d-426c-8198-0ad4a57e7d04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!flux mini submit hostname"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7e4c25e-3ca8-4277-bb70-a0e94bcd223b",
+   "metadata": {},
+   "source": [
+    "`mini submit` admits common options like `--nnodes`, `--ncores`, etc. There are short option equivalents of these options. `--cores-per-task=1` is the default."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "571d8c3d-b24a-415e-b9ac-f58b99a7e92c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!flux mini submit -N1 -n2 sleep inf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cc2bddee-f454-4674-80d4-4a39c5f1bee2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!flux mini submit --help"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac798095",
+   "metadata": {},
+   "source": [
+    "The `flux mini run` command submits a job to Flux (similar to `flux mini submit`) but can attach to the job with `flux job attach`, printing the job's stdout/stderr to the terminal and exiting with the same exit code as the job:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1a516c41-3d8e-46b1-a131-7eef5ff04e69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!flux mini run /bin/false"
```

Comment:
Maybe a better example command would have some `stdout` to demonstrate that output is passed to/from the command?
```suggestion
    "!flux mini run hostname"
```

Reply:
Maybe follow up with - if the job exits with a non-zero exit code, then this will be reported by `flux job attach`
```
"!flux mini run /bin/false"

```